### ISSUE TITLE
Align terminology and concepts with 06 draft?

### ISSUE THREAD

**Issue by aj-stein-nist** [scitt-community/scitt-web#54]
Having reviewed current pages and [the current IETF draft (`-06`)](https://datatracker.ietf.org/doc/html/draft-ietf-scitt-architecture-06), it seems a lot of the terms and maybe key architectural elements have changed enough where the website is out of alignment. Is it time to review and align the site with this draft? Wait for `-07` and last call and final adoption?

**Comment by SteveLasker** [comment]
Yes, definitely, thanks @aj-stein-nist . Although, I'd also say we should now target the pending 07, based on what's in https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/
